doc/md/getting-started/dev-env.md

```diff
@@ -111,3 +113,32 @@ For example, to use base library version `0.9.0`, use the command:
 ```
 mops add base@0.9.0 && mops install
 ```
+
+### Specifying a custom version of `dfx`
+
+To specify a custom version of `dfx`, you can use the [`dfxvm` tool](/docs/current/developer-docs/developer-tools/cli-tools/dfxvm/docs/cli-reference/dfxvm/dfxvm-default). To set a default `dfx` version to be used in your project, run the command:
+
+```
+$ dfxvm default 0.7.2
+...
+info: installed dfx 0.7.2
+info: set default version to dfx 0.7.2
+```
+
+### Pass flags to `moc` in `dfx.json`
+
+You can pass flags directly to `moc` by adding an `args` field in your Motoko canister's description in your project's `dfx.json` file:
+
+Here is an example `dfx.json` canister configuration that uses `args`:
+
+```json
+...
+  "canisters": {
+    "hello": {
+      "type": "motoko",
+      "main": "src/hello/main.mo",
+      "args": "-v --compacting-gc -no-check-ir"
```

Comment:
```suggestion
      "args": "-v --incremental-gc"
```

Reply:
Maybe, we could encourage to use the new GC :).
I am not sure whether `-no-check-ir` is really useful/visible for the users.